### DISCUSSION TITLE
Use the script offline

### DISCUSSION THREAD

**Discussion by GraphiPerso** [linziyou0601/yugioh-card-maker#3] [Q&A]
Hi! 

I search a lot to find how can I start the index.vue on chrome but I didn't understand...

Could you explain me how can I do that please?

Thanks!

**Comment by linziyou0601** [comment]
You need to download and install the nodeJS and npm on your computer first. Then open the cmd (if you use Windows) and use `cd` command change to the project directory.
Finally, enter the command `npm run dev` . You may see the prompt that the website is hosted on http://localhost:3000
So you can open chrome browser and go to http://localhost:3000

For more information, please refer to the document at https://nuxtjs.org/

**Reply by GraphiPerso** [reply]
I install node.js that include npm. I open the command and use cd with the complete parent directory of the script (cd C:\................\yugioh-card-maker-master) then use npm run dev and go to localhost 3000 ... but nothing...

it said that 'nuxt' isn't recognized as a command ....

**Reply by linziyou0601** [reply]
Oh! I forgot one thing, you need to execute `npm install` to install related modules (including nuxt) first. Then you can execute `npm run dev`

**Reply by GraphiPerso** [reply]
it tell me that : 
`126 packages are looking for funding                                                                                      run `npm fund` for details                                                                                                                                                                                                                    21 vulnerabilities (11 moderate, 10 high)                                                                                                                                                                                                       To address issues that do not require attention, run:                                                                     npm audit fix                                                                                                                                                                                                                                 To address all issues (including breaking changes), run:                                                                  npm audit fix --force                                                                                                                                                                                                                         Run `npm audit` for details.                                                                                                                                                                                                                    `

i run it then it says : 
> yugioh_card_maker@1.0.0 dev
> nuxt

I try http://localhost:3000 on chrome and it says "ERR_CONNECTION_REFUSED"

**Reply by linziyou0601** [reply]
You can just skip vulnerabilities message under the testing and I have never encountered ERR_CONNECTION_REFUSED, maybe you can google it.
Basically, if you finish running `npm run dev` and it shows `Listening: http://localhost:3000/`, you should be able to connect to it.

**Reply by GraphiPerso** [reply]
So... it works! Finally! 

Why it didn't work? I didn't think that I must answer the question "are you insterested in participating?" to have the message "listening localhost" 

Thanks a lot for your help! ^^

**Comment by GraphiPerso** [comment]
I try to add french language. I succeed to add all traduction but not the card language and it don't change automaticly to french card language too... 

could you tell me how exactly add a language please? 
How to add custom picture for attribute and spell/trap?
And also how to send to you the file when it will be translate?

**Reply by GraphiPerso** [reply]
I understand now how to add the language and switch automacticly on the good card language... but now there is no more picture :s

**Reply by linziyou0601** [reply]
This is normal, because I only set the card language for Chinese, Japanese, and English. It can be regarded as hard-coded.

But I just Googled it, the card fonts in European languages (or the entire Latin system) and English are the same, so I might be able to rewrite the code to share the card images with the English version and let people add other Latin card languages.

It may take me some time to rewrite the code.

**Reply by GraphiPerso** [reply]
I already rewrite it so I have pictures, and also language ^^ 

just need to verify all to be sure that the whole translation is correct!

**Reply by linziyou0601** [reply]
Okay, and I will also rewrite a version that is more flexible to change the language of the card

**Reply by GraphiPerso** [reply]
I have an issue... I change everything by copy/paste "en" and change according to french language but... 

trap and spell card are displayed weirdly on the card... on English we see [Trap Card] bold and big and in french it's (Carte Piège) not bold and tiny ... why [ become ( ?

**Reply by GraphiPerso** [reply]
OK I understand... I basically change all "en" to "fr" but the font name can be "en" too ... I change it back and it work ^^

**Reply by linziyou0601** [reply]
I've updated the card language file and the structure of the main code, please refer to the readme for details
